static.conf

```diff
@@ -118,6 +118,51 @@ host curslack1-nw {
 	fixed-address 10.112.0.37;
 }
 
+host weidenbaum-AP1 {
+	hardware ethernet 60:e3:27:9f:59:26;
+	fixed-address 10.112.0.38;
+}
+
+host weidenbaum-AP2 {
+	hardware ethernet 60:e3:27:9f:4f:8a;
+	fixed-address 10.112.0.39;
+}
+
+host weidenbaum-AP3 {
+	hardware ethernet 60:e3:27:9f:50:9a;
+	fixed-address 10.112.0.40;
+}
+
+host WR840-AP1 {
+	hardware ethernet a4:2b:b0:eb:01:4a;
+	fixed-address 10.112.0.41;
+}
+
+host WR840-AP2 {
+	hardware ethernet a4:2b:b0:eb:01:3c;
+	fixed-address 10.112.0.42;
```

Comment:
Moin,
die IP ist schon in [Nutzung](https://github.com/freifunkhamburg/dhcp-static/blob/master/static.conf#L123) (tut mir leid, dass ich mich da nicht an die Reihenfolge gehalten habe).
Dein HEAD scheint nicht up-to-date zusein. Rebase doch bitte auf den aktuellen Master (notfalls tuts auch ein merge) und nutze am besten einen Block ab ``10.112.0.43``.
Viele Grüße,
Freddy